tests/unit/test_column_noise.py

```diff
@@ -936,3 +964,7 @@ def test_seeds_behave_as_expected(noise_type, data_col, dataset, dataset_col, du
     ].reset_index(drop=True)
     shortest = min(len(noised), len(noised_different_seed))
     assert (noised.iloc[:shortest] != noised_different_seed.iloc[:shortest]).any()
+
+
+def np_isclose_wrapper(actual_noise, expected_noise, rtol):
+    return np.isclose(actual_noise, expected_noise, rtol)
```

Comment:
Since you're always using the same failure message, you should define that in this function so you don't have to duplicate it each time you call this. I was envisioning this function looking like this:

```
def assert_is_close(actual_noise: float, expected_noise: float, rtol: float) -> None:
    assert np_isclose_wrapper(
        actual_noise, expected_noise, rtol
    ), f"Actual noise is {actual_noise} while expected noise was {expected_noise} with a rtol of {rtol}"
```

Reply:
I see.